.github/workflows/dapp.yml

```diff
@@ -0,0 +1,98 @@
+name: Dapp Tests
+
+on:
+  pull_request:
+    branches: [main, release/**]
+  push:
+    branches: [main, release/**]
+    tags: [v*]
+
+jobs:
+  setup-local-hedera:
+    name: Dapp Tests
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    env:
+      DISPLAY: :99
+      QT_X11_NO_MITSHM: 1
+      _X11_NO_MITSHM: 1
+      _MITSHM: 0
+      DBUS_SESSION_BUS_ADDRESS: /dev/null
+    steps:
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+
+      - name: Setup xvfb (Linux)
+        run: |
+          sudo apt-get install -y libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
+          Xvfb -screen 0 1024x768x24 :99 &
+
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Install packages
+        run: npm ci
+
+      - name: Create .env file
+        run: |
+          cp ./packages/server/tests/localAcceptance.env ./dapp-example/.env
+          cp ./packages/server/tests/localAcceptance.env .env
+
+      - name: Lerna Bootstrap
+        run: npm run setup
+
+      - name: Install pnpm
+        run: npm install -g pnpm
+
+      - name: Build Typescript
+        run: npx lerna run build
+
+      # - name: Run Local Node
+      #   run: npx hedera-local restart
+      #   env:
+      #     NETWORK_NODE_IMAGE_TAG': "0.26.2"
+      #     HAVEGED_IMAGE_TAG': "0.26.2"
+      #     MIRROR_IMAGE_TAG': "0.58.0"
+
+      - name: Run RPC Server
```

Comment:
Please make sure README reflect the right flow to get things working.
Noting just in case you added anything here that's not there to run locally